src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -201,8 +201,11 @@ extension ExposureDetectionViewController {
 			checkButton.setTitle(AppStrings.ExposureDetection.buttonEnable, for: .normal)
 			return
 		}
-		
-		switch state.detectionMode {
+
+		var mode = state.detectionMode
+		if .unknownOutdated == state.risk?.level { mode = .manual }
```

Comment:
This looks like to be unnecessary. This state should never happen bar bugs in background task execution.

Reply:
Technically I agree, but this was the requirement I got. If for whatever reason the automatic update does not work the use needs a way of manually triggering the update.